src/content-fusion/content-fusion.ts

```diff
@@ -19,6 +22,11 @@ export class ContentFusion {
         const localization = res.localization[locale];
         res.localization = {};
         res.localization[locale] = localization;
+      } else {
+        throw <HttpClientProxyError>{
+          message: 'Request failed with status code 404',
```

Comment:
@JERILJOSE96 @iapain I'm not sure this is a correct behaviour of the client
Anyway, if it is there should be some info in the thrown object to specify why there's 404.

Reply:
@espresse I was expecting `axios` will throw HTTPException by itself.